Automattic/newspack-theme
# fix: update Spanish .mo file for updated translations

### All Submissions:


### Changes proposed in this Pull Request:

This PR goes with the updated Spanish translation in #1579 - it generates the .mo file, so the new translation will be picked up by the site. 

### How to test the changes in this Pull Request:

1. Create a post and add at least two authors. 
2. Navigate to WP Admin > Settings > General, and change your site's language to Spanish.
3. View your post with two authors; note that the 'and' in the byline is not translated:

![image](https://user-images.githubusercontent.com/177561/142338313-176550a1-60a0-426f-a279-fb706f40201a.png)

4. Apply the PR.
5. Confirm that the byline is now picking up the new translation:

![image](https://user-images.githubusercontent.com/177561/142338244-94eceda0-4743-45f0-8730-2fae2ec6ae42.png)

### Other information:
